prefect_shell/utils.py

```diff
@@ -76,8 +76,12 @@ async def shell_run_command(
                 stderr = "\n".join(
                     [text async for text in TextReceiveStream(process.stderr)]
                 )
-                logger.error(stderr)
-                msg = f"Command failed with exit code {process.returncode}"
+                if not stderr and lines:
+                    stderr = f"{lines[-1]}\n"
+                msg = (
+                    f"Command failed with exit code {process.returncode}:\n" f"{stderr}"
+                )
+                logger.error(msg)
```

Comment:
I tested and can see that the message in the `RuntimeError` does show up in the UI:
![Screen Shot 2022-07-11 at 9 57 46 AM](https://user-images.githubusercontent.com/12350579/178294175-a147c8f8-befe-4af2-b941-cdcf98bb897d.png)

The `logger.error` call results in the same message being logged twice, so I'd recommend removing this `logger.error` call.